kuma/core/models.py

```diff
@@ -34,6 +38,22 @@ def update(self, **kw):
                                           created=False)
 
 
+class IPBan(models.Model):
+    ip = models.GenericIPAddressField()
+    created = models.DateTimeField(default=timezone.now, db_index=True)
+    deleted = models.DateTimeField(null=True, blank=True)
+
+    objects = IPBanManager()
+
+    def delete(self, *args, **kwargs):
+        self.deleted = timezone.now()
```

Comment:
Hm, I haven't looked for real. If that's not the way you wanna go now (we can do that later (soon)), can you add the deleted field to the admin list display?